charts/tfy-karpenter-config/values.yaml

```diff
@@ -227,6 +227,17 @@ karpenter:
           echo "CONTAINERD_VERSION is empty or not within the specified range."
       fi
 
+      rmmod nvidia_drm
+      rmmod nvidia_modeset
+      rmmod nvidia_uvm
+      rmmod nvidia
```

Comment:
This is on a best-effort basis which is why this section is at the end of the script. `rmmod` for a module fails if some processes are actively using the module or any of its dependents have not been unloaded before.